os/binfmt/binfmt_execmodule.c

```diff
@@ -150,18 +150,11 @@ static void exec_ctors(FAR void *arg)
 int exec_module(FAR struct binary_s *binp)
 {
 	FAR struct task_tcb_s *tcb;
```

Comment:
How about renaming this to distinguish rtcb?
loading_stcb? (s : start)
main_tcb? mtcb?

Reply:
Ok. I will rename it as newtcb.